wordpress-mobile/WordPress-Android
# Introduce activity log filters feature flag

Related to #13268 

This PR introduces a local feature flag for the Activity Log filters feature. When the feature flag is enabled a "Date range" chips is shown in the activity log.

Note: It doesn't matter if it gets merged into 16.1 or 16.2.

To test:
1. Open Activity Log and make sure the "Date range" chip is hidden
2. My Site -> Avatar icon in the toolbar -> App Settings -> Test feature configuration
3. Enable ActivityLogFiltersFeatureConfig and click on "Restart app"
4. Go back to Activity Log and notice "Date range" chip is shown

PR submission checklist:
